sources/lib/DependencyInjection/Configuration.php

```diff
@@ -42,7 +42,8 @@ public function getConfigTreeBuilder()
                     ->prototype('array')
                         ->children()
                             ->scalarNode('dsn')->isRequired()->end()
-                            ->scalarNode('class:session_builder')->defaultValue('\PommProject\ModelManager\SessionBuilder')->end()
+                            ->scalarNode('class:session_builder')->defaultNull()->end()
```

Comment:
if it does not work anymore, you should trigger a deprecation warning when it is set to a value